compiler/src/dmd/dsymbolsem.d

```diff
@@ -835,7 +835,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 }
                 if (!isWild)
                 {
-                    dsym.error("`inout` variables can only be declared inside `inout` functions");
+                    dsym.error(": `inout` variables can only be declared inside `inout` functions");
```

Comment:
You could also use the global `.error` function and pass it `dsym.toChars()` so you can put the name in a more convenient part of the message

Reply:
I think there are a lot of these still to fix (for another pull), so I'd rather keep the changes simple.

Reply:
I think there are a lot of these still to fix (for another pull), so I'd rather keep the changes simple.